pymc3/variational/opvi.py

```diff
@@ -478,44 +478,155 @@ def from_function(cls, f):
+        as conditional distribution. You an read more about in the corresponding paper
+        mentioned in references
 
-class Group(object):
-    """
-    Grouped Approximation that is used for modelling mutual dependencies
-    for a specified group of variables. Base for local and global group
+        Batched mode is a special case approximation that treats every 'row', of a tensor as
+        independent from each other. Some distributions can't do that by
+        definition e.g. :class:`Empirical` that consists of particles only.
+
+    -   **has_logq**
+        Tells that distribution is defined explicitly
+
+    These constants help providing the correct inference method for given parametrization
+
+    Examples
+    --------
+
+    Basic Initialization
+    ^^^^^^^^^^^^^^^^^^^^
+    :class:`Group` is a factory class. You do not need to call every ApproximationGroup explicitly.
+    Passing the correct `vfam` (*v*ariational *fam*ily) argument you'll tell what
+    parametrization is desired for the group. This helps not to overload code with lot's of classes.
+
+    >>> group = Group([latent1, latent2], vfam='mean_field')
+
+    The other way to select approximation is to provide `params` dictionary that has some
+    predefined well shaped parameters. Keys of the dict serve as an identifier for variational family and help
+    to autoselect the correct group class. To identify what approximation to use, params dict should
+    have the full set of needed parameters. As there are 2 ways to instantiate the :class:`Group`
+    passing both `vfam` and `params` is prohibited. Partial parametrization is prohibited by design to
+    avoid corner cases and possible problems.
+
+    >>> group = Group([latent3], params=dict(mu=my_mu, rho=my_rho))
+
+    Important to note that in case you pass custom params they will not be autocollected by optimizer, you'll
+    have to provide them with `more_obj_params` keyword.
+
+    Using AEVB
+    ^^^^^^^^^^
+    Autoencoding variational Bayes is a powerful tool to get conditional :math:`q(\lambda|X)` distribution
+    on latent variables. It is well supported by PyMC3 and all you need is to provide a dictionary
+    with well shaped variational parameters, the correct approximation will be autoselected as mentioned
+    in section above. However we have some implementation restrictions in AEVB. They require autoencoded
+    variable to have first dimension as *batch* dimension and other dimensions should stay fixed.
+    With this assumptions it is possible to generalize all variational approximation families as
+    batched approximations that have flexible parameters and leading axis.
+
+    Only single variable local group is supported. Params are required.
+
+    for mean field
+    >>> group = Group([latent3], params=dict(mu=my_mu, rho=my_rho), local=True)
+    or for full rank
+    >>> group = Group([latent3], params=dict(mu=my_mu, L_tril=my_L_tril), local=True)
```

Comment:
`my_mu` and` my_rho` are usually estimated with neural network or function approximator. I think it should be explicitly mentioned here or you can point to some example notebook.